cmd/kyma/provision/minikube/cmd.go

```diff
@@ -19,11 +19,13 @@ import (
 )
 
 const (
-	kubernetesVersion  string = "1.14.6"
-	bootstrapper       string = "kubeadm"
-	vmDriverHyperkit   string = "hyperkit"
-	vmDriverHyperv     string = "hyperv"
-	vmDriverNone       string = "none"
+	kubernetesVersion string = "1.14.6"
+	bootstrapper      string = "kubeadm"
+	vmDriverHyperkit  string = "hyperkit"
+	vmDriverHyperv    string = "hyperv"
+	vmDriverNone      string = "none"
+	// it's used in cfg_windows.go
+	// nolint: deadcode,varcheck
```

Comment:
Instead of adding this, a better way to solve this warning would be to use this constant in line 35 of this same file (were currently `virtualbox` is a string literal).

Reply:
agree 👍